server/views/pages/calculation/reason.njk

```diff
@@ -11,8 +11,8 @@
 
 {% block beforeContent %}
     {{ super() }}
-    {% if model.dpsEntryPoint %}
-        {% set backlink = "/?prisonId=" + model.prisonerDetail.offenderNo %}
+    {% if model.dpsEntryPoint or featureToggles.useCCARDLayout %}
```

Comment:
I was just looking at this. `model.dpsEntryPoint` isn't actually set so I don't think this has ever worked! At least this will fix it forever once CCARD is released.